pkg/inject/envoy.go

```diff
@@ -18,6 +18,8 @@ type EnvoyTemplateVariables struct {
 	MeshName                     string
 	VirtualNodeName              string
 	Preview                      string
+	EnableSDS                    bool
```

Comment:
Interesting, this is showing up as diff, when this change is already in master. 
https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/pkg/inject/envoy.go#L126

Reply:
Check the PR instead of the last commit - that will show the accurate diff to the main branch